pkg/utils/server.go

```diff
@@ -145,3 +146,12 @@ func spdkMockServerCommunicate(rpc spdk.JSONRPC, l net.Listener, toSend []string
 func ResourceIDToVolumeName(resourceID string) string {
 	return fmt.Sprintf("//storage.opiproject.org/volumes/%s", resourceID)
 }
+
+// OpiAdressFamilyToSpdk converts opi address family to the one used in spdk
+func OpiAdressFamilyToSpdk(adrfam pb.NvmeAddressFamily) string {
+	if adrfam == pb.NvmeAddressFamily_NVME_ADDRESS_FAMILY_UNSPECIFIED {
+		return ""
+	}
```

Comment:
ok for now, maybe better to have explicit switch case for translation between OPI PROTO and SPDK names...